src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/ResponseExceptionHandler.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+import static org.springframework.http.ResponseEntity.status;
+
+@ControllerAdvice
+public class ResponseExceptionHandler extends ResponseEntityExceptionHandler {
+
+    private static final Logger log = LoggerFactory.getLogger(ResponseExceptionHandler.class);
+
+    @ExceptionHandler(Exception.class)
+    protected ResponseEntity<ErrorResponse> handleInternalException(Exception exception) {
+        log.error(exception.getMessage(), exception);
+        return status(INTERNAL_SERVER_ERROR).body(new ErrorResponse(exception.getMessage()));
```

Comment:
Some existing tests (for other endpoints) assume 500 responses have body.

Reply:
I think it's best to either return a generic failure message or not return anything at all (and adjust tests). The caller shouldn't be informed about the details of our internal errors.

Reply:
As discussed, this PR doesn't change the behaviour, as far as the response body is concerned. This can be addressed at a different point.